### ISSUE TITLE
Mobile data connectivity icons

### ISSUE THREAD

**Issue by sweeneyuk** [MohammadAG/Xposed-Tinted-Status-Bar#91]
Hi on note 3 n9005 using stock rom the mobile data activity icons do not theme to black on light status bar yet wifi icons do. 

Thanks for your help and brilliant work. 

![screenshots_2014-07-27-21-10-50](https://cloud.githubusercontent.com/assets/8141896/3715094/65bf4296-15cf-11e4-9ce3-76d0f11c38e2.png)

**Comment by AbFarid** [comment]
Yep, same here. Btw, you can hide the Smart Stay icon via Wanam Xposed module.  And I was wondering how did you get your circular battery to appear tinted, mine anyways stays white.

**Comment by sweeneyuk** [comment]
If you use gravity box to change your battery icon and enable to colour
status bar icons it should tint.

**Comment by AbFarid** [comment]
Oh yeah, it works, thanks!
